src/app/services/nearbyCities.service.js

```diff
@@ -20,7 +21,7 @@ export default class NearbyCitiesService {
                 return self.getGeoNames(lat, lon);
             })
             .catch(response => {
-                console.error(response.data.message);
+                throw response
```

Comment:
if you are just throwing the error then doesn't make sense to catch it in the first place